coil-base/src/main/java/coil/transition/CrossfadeTransition.kt

```diff
@@ -42,7 +41,7 @@ class CrossfadeTransition(
 
         // Don't animate if the view is not visible as CrossfadeDrawable.onDraw
```

Comment:
You may want to update the comment line, as well.

```suggestion
        // Don't animate if the view is not shown as CrossfadeDrawable.onDraw
```

Reply:
Ah good point.